src/fusco.erl

```diff
@@ -504,15 +532,16 @@ is_ipv6_host(Host) ->
                 {ok, _} ->
                     false;
                 _ ->
-                    case inet:getaddr(Host, inet6) of
-                        {ok, _} ->
-                            true;
-                        _ ->
-                            false
-                    end
+                    inet_getaddr(inet:getaddr(Host, inet6))
             end
     end.
 
+%%------------------------------------------------------------------------------
+%% @private
+%%------------------------------------------------------------------------------
+inet_getaddr({ok, _}) -> true;
```

Comment:
This function should be called `is_proper_addr` or `is_ok_response` or something clearer